tracee-ebpf/tracee/tracee.go

```diff
@@ -904,8 +906,9 @@ func (t *Tracee) Run() error {
 	<-sig
 	t.eventsPerfMap.Stop()
 	t.fileWrPerfMap.Stop()
-	t.printer.Epilogue(t.stats)
-
+	if t.config.ChanEvents != nil {
+		t.printer.Epilogue(t.stats)
```

Comment:
I think there is some important information in the epilog that a consumer of tracee as a library may want to know. This information includes, for example, the number of lost events and the event count.

Reply:
Sure but this shouldn't be "printed" for the consumer right? 
We should add another API to retrieve that other information. And also I think we should take all the printing out of Tracee library. 
Anyway, this is just a bug fix which should have been included with the original commit